tests/model_monitoring/test_stream_processing.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+import mlrun
+from mlrun.model_monitoring.stream_processing import EventStreamProcessor
+
+
+@pytest.mark.parametrize("tsdb_connector", ["v3io", "taosws"])
+@pytest.mark.parametrize("endpoint_store", ["v3io", "mysql"])
+def test_plot_monitoring_serving_graph(tsdb_connector, endpoint_store):
+    project_name = "test-stream-processing"
+
+    processor = EventStreamProcessor(
+        project_name,
+        1000,
+        10,
+        "mytarget",
+    )
+
+    fn = mlrun.new_function(
+        kind="serving",
+        name="my-fn",
+    )
```

Comment:
Do we have an updated version of this?
I understood that `mlrun.code_to_function` and `mlrun.new_function` are deprecated.
Just asking, because I also encountered it recently.

Reply:
I'm not aware of anything to that effect. They're not deprecated in the code though.

Reply:
See this: #6235

Reply:
Ok, I changed it :)